src/analyze/analyze.c

```diff
@@ -982,7 +982,7 @@ static int list_dependencies_one(sd_bus *bus, const char *name, unsigned level,
 
         STRV_FOREACH(c, deps) {
                 times = hashmap_get(unit_times_hashmap, *c);
-                if (times_in_range(times, boot) && times->activated >= service_longest)
+                if (times && times_in_range(times, boot) && times->activated >= service_longest)
```

Comment:
times_in_range() checks for NULL… So all three checks that are added here don't matter.

Reply:
Hah, I see, thanks. I'll suppress the warning in these cases.